packages/@aws-cdk/aws-ecs/lib/fargate/fargate-task-definition.ts

```diff
@@ -134,6 +145,31 @@ export class FargateTaskDefinition extends TaskDefinition implements IFargateTas
       throw new Error('Ephemeral storage size must be between 21GiB and 200GiB');
     }
 
+    if (props.runtimePlatform?.operatingSystemFamily) {
+      if (props.runtimePlatform?.operatingSystemFamily?._operatingSystemFamily !== OperatingSystemFamily.LINUX._operatingSystemFamily) {
```

Comment:
This makes an assumption that all non-Linux operatingSystemFamily values will all be windows. Is that the case? Or would it be possible that some non-Linux and non-Windows operatingSystemFamily values will be added to this? 

I think we probably need to check that the operatingSystemFamily is a WINDOWS_XXX one, rather than checking that it is *not* LINUX.